sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -285,22 +278,20 @@ namespace Azure { namespace Core { namespace Http {
      * @brief Set URL path.
      *
      * @param path URL path.
```

Comment:
Same here/elsewhere. Make sure docs, src, and header match in terms of parameter names.